modules/schematics-core/utility/find-module.ts

```diff
@@ -57,7 +57,7 @@ export function findModuleFromOptions(
     } else if (host.exists(modulePath + '/' + moduleBaseName + '.module.ts')) {
       return normalize(modulePath + '/' + moduleBaseName + '.module.ts');
     } else {
-      throw new Error('Specified module does not exist');
+      throw new Error(`Specified module path ${modulePath} does not exist`);
```

Comment:
Run `yarn copy:schematics` after modifying files in the `schematics-core` folder so the changes are copied to the other packages.

Reply:
I understand and additionally commit its changes.
Please continue your review.